CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.2.22.1]
```

Comment:
@ameyapat, could you confirm this will work with our scripts?

Reply:
@oldalton @Veena11 This does not follow semantic versioning: https://semver.org/ and will not work with our script in release pipeline. You can upgrade the patch version to 23 or define a pre-release version as 1.2.22-a

Reply:
updated to 1.2.23